core/services/synchronization/metrics.go

```diff
@@ -0,0 +1,34 @@
+package synchronization
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
```

Comment:
You can have one large `var` with all these variables instead of having multiple smaller ones :P